Source/WebKit/WebProcess/WebPage/Cocoa/WebPageCocoa.mm

```diff
@@ -146,7 +146,7 @@
 #if ENABLE(PDF_PLUGIN)
 bool WebPage::shouldUsePDFPlugin(const String& contentType, StringView path) const
 {
-    return pdfPluginEnabled()
+    auto shouldUsePlugin = pdfPluginEnabled()
```

Comment:
`bool` is exactly as long as `auto` and it clearer.

Reply:
Agreed. I'll change it. Just a mechanical habit to use `auto`

Reply:
https://github.com/WebKit/WebKit/commit/34f1fc9ef36a4c6d5f5e333cf00080db7b704674